lua/null-ls/builtins/diagnostics/phpcs.lua

```diff
@@ -28,7 +28,8 @@ return h.make_builtin({
             "1",
             -- process stdin
             "--stdin-path=$FILENAME",
-            "-",
+            -- get absolute paths under params.output.files
+            "--basepath=",
```

Comment:
I've tried using
```suggestion
            '--basepath=""',
```

but that seems to result in the tool looking for a directory literally named `""`